### ISSUE TITLE
support question: What are the negatives/gotchas with the pattern lab approach?

### ISSUE THREAD

**Issue by ElijahLynn** [phase2/particle#296]
When developing the front end & UX with this method, what are the gotchas in doing so vs. using straight Twig from D8? I understand the benefit quite well but just want to know what to look out for.

**Comment by illepic** [comment]
If your use-case requires a design system that is as agnostic from its consuming application as possible, Particle will suit your needs.

There is nothing wrong with writing Twig files for a Drupal theme. Much of the client work we find ourselves in requires building out much, much more than just "Drupal theme" and we prefer to see components prototyped out looooooong before we need to build an entire Drupal site and content model.

Basically, if you've ever had to ask yourself "Where in the hell on this dev Drupal site can I go to see the 7 variations of the 'card' that the designers drew up in Sketch?" then maybe a vanilla Drupal theme is not the answer. Maybe more tooling configured together to build a design system outside of Drupal is what you're looking for.

**Comment by illepic** [comment]
@ElijahLynn Imma close this unless we have any more bikeshedding we want to do ;)
